inference-engine/tests/unit/inference_engine/ie_blob_proxy.cpp

```diff
@@ -17,40 +16,24 @@ using namespace InferenceEngine::details;
 
 #define MAKE_SHORT(l, h) h * 0x100 + l
 
-class BlobProxyTests: public ::testing::Test {
-protected:
-    virtual void TearDown() {
-    }
-
-    virtual void SetUp() {
-    }
+using BlobProxyTests = ::testing::Test;
```

Comment:
Even this `using` is not required to create a test entity, but let's keep it for the future. Take a look at [Simple Test creation](https://github.com/google/googletest/blob/master/googletest/docs/primer.md#simple-tests) in test - first argument in the macros is just test suite name and haven't to any c++ object.